src/unix/proctitle.c

```diff
@@ -83,15 +81,11 @@ char** uv_setup_args(int argc, char** argv) {
 
 
 int uv_set_process_title(const char* title) {
-  char* new_title;
-  /* Copy the title into our own buffer. We don't want to free the pointer
-   * on libuv shutdown because the program might still be using it. */
-  new_title = uv__strdup(title);
-  if (new_title == NULL)
-    return -ENOMEM;
-  uv__free(process_title.str);
-  process_title.str = new_title;
-  process_title.len = strlen(new_title);
+  if (process_title.len == 0)
+    return 0;
```

Comment:
You're right, but I don't think it makes a lot of sense to call `prctl` but not change `argv[0]`. IMO it should either work entirely or not work at all (or maybe return an error, but certainly not fail silently). And BTW, this commit just reverts the previous changes I made to this file, but I'd be happy to touch it up a little if need be.